roles/kubernetes/master/tasks/kubeadm-etcd.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Calculate etcd cert serial
```

Comment:
why not move those 2 tasks to seperate file and then use include tasks with var?

Reply:
That would make sense to do, and easier to maintain

Reply:
Can we do that in a followup?